hack/tidy-check.sh

```diff
@@ -27,7 +31,7 @@ TMP_DIR="$THIS_DIR/.tmp.tidy-check"
 TMP_MOD_FILE="$TMP_DIR/go.mod"
 TMP_SUM_FILE="$TMP_DIR/go.sum"
 TARGET_GO_VERSION="1.13"
-TARGET_GO_VERSION_PATTERN="go$TARGET_GO_VERSION.*"
+TARGET_GO_VERSION_PATTERN="go$TARGET_GO_VERSION*"
```

Comment:
out of curiosity, why "." needs to be deleted?

Reply:
Because it could be go1.13 or go1.13.X.

Reply:
Got it, now I see go uses 1.X instead of 1.X.0 for the first patch releases.